Ruslan Gaiazov/script.js

```diff
@@ -0,0 +1,99 @@
+window.onload = function() {
+  var addTask = document.getElementById("addTask-button");
+  var textTask = document.getElementById("textTask");
+  var deadlineTask = document.getElementById("deadlineTask");
+  var filterAllTasks = document.getElementById("filter-all");
+  var filterDone = document.getElementById("filter-done");
+  var filterNotDone = document.getElementById("filter-notdone");

```

Comment:
Из придирок :) так было бы получше:
```suggestion
  var filterNotDone = document.getElementById("filter-in-progress");
```